docs/devupdate.md

```diff
@@ -289,3 +289,17 @@ public function add_sticky_action_buttons(
     ?string $submitlabel = null,
 );
 ```
+
+A new method `filter_shown_headers() is created to [Forms API](./apis/subsystems/form/index.md#filter_shown_headers) to show some expanded headers only and hide the rest.
```

Comment:
```suggestion
### filter_shown_headers

A new method `filter_shown_headers()` is created to [Forms API](./apis/subsystems/form/index.md#filter_shown_headers) to show some expanded headers only and hide the rest.
```

Reply:
I would suggest adding a header also for the add_sticky_action_buttons to make it easier to find them in the index too.